doc/index.md

```diff
@@ -456,6 +456,19 @@ Returns: a Promise that is fulled one the web socket is connected
   and other necessary async tasks are complete.
 
 
+#### `startSoftphoneSesssion(softphoneSessionParams: IStartSoftphoneSessionParams)`
+Starts a softphone call session with the given peer or peers.
+
+`initialize()` must be called first.
+
+Declaration:
+``` ts
+startSoftphoneSession(softphoneSessionParams: IStartSoftphoneSessionParams): Promise<any>;
```

Comment:
If we know what the return is going to be, we should use the type here.

Reply:
Added